indy_node/server/domain_req_handler.py

```diff
@@ -499,55 +495,53 @@ def handleGetRevocRegDelta(self, request: Request):
-            encoded_revoc_reg_def = self.state.get_for_root_hash(past_root_to, revoc_reg_def_id)
-            assert encoded_revoc_reg_def
-            revoc_reg_def, _, _ = domain.decode_state_value(encoded_revoc_reg_def)
-            strategy_cls = self.get_revocation_strategy(revoc_reg_def[VALUE][ISSUANCE_TYPE])
-
-            if req_ts_from:
-                past_root_from, reg_entry_from = self._get_reg_entry_by_timestamp(req_ts_from, path_to_reg_entry)
-                req_entry_accum_from, \
-                    seq_no_from, \
-                    last_update_time_from, \
-                    reg_entry_accum_proof_from = self._get_reg_entry_accum_by_timestamp(req_ts_from, path_to_reg_entry_accum)
-                # Compute issued/revoked lists corresponding with ISSUANCE_TYPE strategy
-                result_issued, result_revoked = strategy_cls.get_delta({ISSUED: reg_entry_to[VALUE].get(ISSUED, []),
-                                                                        REVOKED: reg_entry_to[VALUE].get(REVOKED, [])},
-                                                                       {ISSUED: reg_entry_from[VALUE].get(ISSUED, []),
-                                                                        REVOKED: reg_entry_from[VALUE].get(REVOKED, [])})
-            else:
-                result_issued, result_revoked = strategy_cls.get_delta({ISSUED: reg_entry_to[VALUE].get(ISSUED, []),
-                                                                        REVOKED: reg_entry_to[VALUE].get(REVOKED, [])},
-                                                                       None)
-            reply = {
-                REVOC_REG_ID: str(path_to_reg_entry),
-                REVOC_TYPE: revoc_reg_def.get(REVOC_TYPE),
-                VALUE: {
-                    ACCUM_TO: reg_entry_accum_to,
-                    ISSUED: result_issued,
-                    REVOKED: result_revoked
-                }
+            encoded_revoc_reg_def = self.state.get_for_root_hash(past_root_to,
+                                                                 revoc_reg_def_id)
+            if encoded_revoc_reg_def:
+                revoc_reg_def, _, _ = domain.decode_state_value(encoded_revoc_reg_def)
+                strategy_cls = self.get_revocation_strategy(revoc_reg_def[VALUE][ISSUANCE_TYPE])
+
+                if req_ts_from:
+                    past_root_from, reg_entry_from = self._get_reg_entry_by_timestamp(req_ts_from, path_to_reg_entry)
+                    req_entry_accum_from, \
+                        seq_no_from, \
+                        last_update_time_from, \
+                        reg_entry_accum_proof_from = self._get_reg_entry_accum_by_timestamp(req_ts_from, path_to_reg_entry_accum)
+                    # Compute issued/revoked lists corresponding with ISSUANCE_TYPE strategy
+                    result_issued, result_revoked = strategy_cls.get_delta({ISSUED: reg_entry_to[VALUE].get(ISSUED, []),
```

Comment:
Passed objects should be stored in separate variables, it is really hard to read.